src/components/views/messages/MFileBody.tsx

```diff
@@ -105,11 +105,6 @@ export default class MFileBody extends React.Component<IProps, IState> {
     declare public context: React.ContextType<typeof RoomContext>;
 
     public state: IState = {};
-
-    public static defaultProps = {
```

Comment:
Default props would expect `showGenericPlaceholder?: boolean` would cause the signature of MFIleBody to expect a parameter for  `showGenericPlaceholder`, oddly. 

Since this is a single case, it was easier to just default it in the render function.